tests/dummy/app/components/cookbook/paginated-table.js

```diff
@@ -54,7 +54,8 @@ export default class PaginatedTable extends BaseTable {
     }
 
     this.page = page;
-    this.model.clear();
+    const currentRecordsArray = this.table.rows.toArray();
+    this.table.removeRows(currentRecordsArray);
```

Comment:
Would `this.model.toArray()` work as well? Might be closer to the `this.model.clear` we had before.